02_Architecture/05_InterruptHandling.md

```diff
@@ -364,15 +364,15 @@ While some of these vectors are unused, they are still reserved and might be use
 
 - _Page Fault_: Easily the most common one to run into. It means there was an issue with translating a virtual address into a physical one. This does push an error code which describes the memory access that triggered the page fault. Note the error describes what was being attempted, not what caused translation to fail. The `%cr2` register will also contain the virtual address that was being translated. 
 - _General Protection Fault_: A GP fault can come from a large number of places, although it's generally from an instruction dealing with the segment registers in some way. This includes `iret` (it modifies cs/ss), and others like `lidt`/`ltr`. It also pushes an error code, which is described below. A GP fault can also come from trying to execute a privileged instruction outside when it's not allowed to be. This case is different to an undefined opcode, as the instruction exists, but is just not allowed.
-- _Double Fault_: This means something has gone horribly wrong, and the system is not in a state that can be recovered from. Commonly this occurs because the cpu could not call the GP fault handler, but it can be triggered by hardware conditions too. This should considered our last chance to clean up and save any state. If a double fault is not handled, the cpu will 'triple fault', meaning the system resets.
+- _Double Fault_: This means something has gone horribly wrong, and the system is not in a state that can be recovered from. Commonly this occurs because the cpu could not call the GP fault handler, but it can be triggered by hardware conditions too. This should be considered as our last chance to clean up and save any state. If a double fault is not handled, the cpu will 'triple fault', meaning the system resets.
 
-A number of the reserved interrupts will not be fired by default, they require certain flags to be set. For example the x87 fpu error only occurs if `CR0.NE` is set, otherwise the fpu will silently fail. The SIMD error will only occur if the cpu has been told to enable SSE. Others like bound range exceeded or device not available can only occur on specific instructions, and are generally unseen. 
+A number of the reserved interrupts will not be fired by default, they require certain flags to be set. For example the x87 FPU error only occurs if `CR0.NE` is set, otherwise the FPU will silently fail. The SIMD error will only occur if the cpu has been told to enable SSE. Others like bound range exceeded or device not available can only occur on specific instructions, and are generally unseen. 
```

Comment:
can you make them italic too? thanks

Reply:
sure, but why though?

Reply:
i was thinking that since it is an acronym/register name it could make sense to have it in with a different text format.
Btw if you think is not necessar,y it's ok, you can leave it just in capital letters (that is better than all lowercase! :) ) .